runtime/doc/todo.txt

```diff
@@ -2091,14 +2089,6 @@ MSDOS and Win32:
     backslashes. (Ronald Hoellwarth)
 
 
-Windows 95:
-8   Editing a file by its short file name and writing it, makes the long file
-    name disappear.  Setting 'backupcopy' helps.
-    Use FindFirstFile()->cAlternateFileName in fname_case() (George Reilly).
-8   Doing wildcard expansion, will match the short filename, but result in the
-    long filename (both DJGPP and Win32).
-
-
 Win32 console:
 9   When editing a file by its short file name, it should be expanded into its
     long file name, to avoid problems like these: (Mccollister)
```

Comment:
Hi Justin!

On Do, 02 Jul 2015, Justin M. Keyes wrote:

> > @@ -2091,14 +2089,6 @@ MSDOS and Win32:
> >      backslashes. (Ronald Hoellwarth)
> >  
> >  
> > -Windows 95:
> > -8   Editing a file by its short file name and writing it, makes the long file
> > -    name disappear.  Setting 'backupcopy' helps.
> > -    Use FindFirstFile()->cAlternateFileName in fname_case() (George Reilly).
> > -8   Doing wildcard expansion, will match the short filename, but result in the
> > -    long filename (both DJGPP and Win32).
> > -
> > -
> >  Win32 console:
> >  9   When editing a file by its short file name, it should be expanded into its
> >      long file name, to avoid problems like these: (Mccollister)
> 
> We still support `%:8`. Not sure if that is what this is about though.

Are you sure? I believe this was a windows specific so was only 
available in Windows built. And since Windows support was removed, I 
wouldn't bet, that :8 actually does anything currently.

Best,
Christian
-- 
Was die neuen Unwissenden holen müssen:
Frequenzbiegezangen